src/couch_views/test/couch_views_indexer_test.erl

```diff
@@ -32,17 +29,19 @@ indexer_test_() ->
                 foreach,
                 fun foreach_setup/0,
                 fun foreach_teardown/1,
-                ?I_HEART_EUNIT([
-                    fun indexed_empty_db/1,
-                    fun indexed_single_doc/1,
-                    fun updated_docs_are_reindexed/1,
-                    fun updated_docs_without_changes_are_reindexed/1,
-                    fun deleted_docs_not_indexed/1,
-                    fun deleted_docs_are_unindexed/1,
-                    fun multipe_docs_with_same_key/1,
-                    fun multipe_keys_from_same_doc/1,
-                    fun multipe_identical_keys_from_same_doc/1
-                ])
+                [
+                    with([?TDEF(indexed_empty_db)]),
```

Comment:
Can use the new `?TDEF_FE` macro from https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/fabric/test/fabric2_changes_fold_tests.erl#L40 to avoid repeating the `with` line